proposals/mainnet_migration.md

```diff
@@ -0,0 +1,528 @@
+<p>
+This value will be used for every delegate account in the snapshot block.
+   </td>
+  </tr>
+  <tr>
+   <td><code><a href="https://github.com/LiskHQ/lips/blob/main/proposals/lip-0051.md#constants-and-notations">TOKEN_ID_LSK_MAINCHAIN</a></code>
+   </td>
+   <td>object
+   </td>
+   <td><code>{"chainID": 0, "localID": 0}</code>
+   </td>
+   <td>Token ID of the LSK token on mainchain.
+   </td>
+  </tr>
+  <tr>
+   <td><code><a href="https://github.com/LiskHQ/lips/blob/main/proposals/lip-0051.md#constants-and-notations">LOCAL_ID_LSK</a></code>
+   </td>
+   <td>uint32
+   </td>
+   <td>0
+   </td>
+   <td>The local ID of the LSK token on mainchain.
+   </td>
+  </tr>
+  <tr>
+   <td><code>MODULE_ID_DPOS</code>
+   </td>
+   <td>uint32
+   </td>
+   <td>TBD
+   </td>
+   <td>Module ID of the <a href="https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md">DPoS Module</a>
+   </td>
+  </tr>
+  <tr>
+   <td><code>MODULE_ID_AUTH</code>
+   </td>
+   <td>uint32
+   </td>
+   <td>TBD
+   </td>
+   <td>Module ID of the <a href="https://github.com/LiskHQ/lips/blob/main/proposals/lip-0041.md">Auth Module</a>
+   </td>
+  </tr>
+  <tr>
+   <td><code>MODULE_ID_TOKEN</code>
+   </td>
+   <td>uint32
+   </td>
+   <td>TBD
+   </td>
+   <td>Module ID of the <a href="https://github.com/LiskHQ/lips/blob/main/proposals/lip-0051.md">Token Module</a>
+   </td>
+  </tr>
+  <tr>
+   <td><code>MODULE_ID_LEGACY</code>
+   </td>
+   <td>uint32
+   </td>
+   <td>TBD
+   </td>
+   <td>Module ID of the <a href="https://github.com/LiskHQ/lips/blob/main/proposals/lip-0050.md">Legacy Module</a>.
+   </td>
+  </tr>
+  <tr>
+   <td><code>INVALID_ED25519_KEY</code>
+   </td>
+   <td>bytes
+   </td>
+   <td>
+
+```   
+0x7fffffff
+ffffffffff
+ffffffffff
+ffffffffff
+ffffffffff
+ffffffffff
+ffffff
+```
+
+   </td>
+   <td>An Ed25519 public key for which the signature validation always fails. This value is used for the <code>generatorPublicKey</code> property of validators in the snapshot block for which no public key is contained within the history since the last snapshot block.
+   </td>
+  </tr>
+  <tr>
+   <td><code>DPOS_INIT_ROUNDS</code>
+   </td>
+   <td>uint32
+   </td>
+   <td>60,480
+   </td>
+   <td>The number of rounds for the bootstrap period following the snapshot block. This number corresponds to one week assuming no missed blocks.
+   </td>
+  </tr>
+  <tr>
+   <td><code>HEIGHT_SNAPSHOT</code>
+   </td>
+   <td>uint32
+   </td>
+   <td>TBD
+   </td>
+   <td>The height of the block from which a state snapshot is taken. This block must be an end of a round block. The snapshot block has then the height <code>HEIGHT_SNAPSHOT +1.</code>
```

Comment:
```suggestion
   <td>The height of the block from which a state snapshot is taken. This block must be an end of a round block. The snapshot block has then the height `HEIGHT_SNAPSHOT +1`.
```

Reply:
I think the conversion tool converts tables with html tags rather than pure markdown tables. All good, I resolve it.